backend/resolvers/Mutation/port.schema.json

```diff
@@ -29,11 +29,6 @@
         "Tietojenkäsittelytieteen kandiohjelma"
       ]
     },
-    "defaultCourse": {
-      "type": "string",
-      "description": "Name of course to view by default. Must be defined in courses.",
-      "examples": ["Ohjelmoinnin perusteet"]
-    },
```

Comment:
This should be kept so the defaultCourse field is just ignored to stay compatible with old exported workspaces